tests/expected_metrics/metrics_test_ote_training.yml

```diff
@@ -0,0 +1,21 @@
+'ACTION-training_evaluation,model-gen3_mobilenetV2_ATSS,dataset-bbcd,num_iters-KEEP_CONFIG_FIELD_VALUE,batch-KEEP_CONFIG_FIELD_VALUE,usecase-reallife':
+        'metrics.accuracy.f-measure':
+                'target_value': 0.81
+                'max_diff_if_less_threshold': 0.005
+                'max_diff_if_greater_threshold': 0.015
```

Comment:
What does that mean?

Reply:
My guess is that this is equivalent to:
```
-max_diff_if_less_threshold < actual - expected < max_diff_if_greater_threshold
```
@LeonidBeynenson, right?

Reply:
Sorry that did not answer.
Yes, you are right: if both of the values `max_diff_if_less_threshold` and `max_diff_if_greater_threshold` are set, then it works as you wrote.
But we can point only one of the values -- in this case the range of accessible values will be half-bounded.

Reply:
I just wanted to emphasize, that IMO naming is not perfect, at least.

Reply:
Ohh, Pavel, I agree with you.
If you have ideas how to call them, please, propose your version!